geps/gep-1713/index.md

```diff
@@ -74,7 +74,7 @@ type ListenerNamespaces struct {
 	// * None: Only listeners defined in the Gateway's spec are allowed
 	//
 	// +optional
-	// +kubebuilder:default=Same
```

Comment:
This PR seems to jump us all the way from `provisional` to `experimental` without hitting `implementable`. If we have consensus then there might not actually be anything wrong with that, but bare minimum we should update the GEP status accordingly here.

Reply:
Want me to split out the APIs in to a separate PR?

Reply:
No that's OK! I just think we should flag the status change here and then please feel free to consider this comment resolved.

I just realized another opportunity to update the GEP, but I'll create a separate comment.